app/index.js

```diff
@@ -155,33 +168,46 @@ clock.ontick = (evt) => {
     dateLabel.text = `${weekday}, ${month} ${day}`;
 
     // Steps
-    steps.text = `${(Math.floor(today.adjusted.steps / 1000) || 0)}k`;
-    if (steps.text >= (goals.steps || 0)) {
-        steps.style.fill = 'fb-green'; //green
-    } else if (steps.text >= (goals.steps || 0) / 2) {
-        steps.style.fill = 'fb-peach'; //yellow
-    } else {
-        steps.style.fill = 'fb-orange'; //pink
+    if (today.adjusted.steps > 0) {
+        try {
+            steps.text = `${(Math.floor(today.adjusted.steps / 1000) || 0)}k`;
+            if (steps.text >= (goals.steps || 0)) {
+                steps.style.fill = 'fb-green'; //green
+            } else if (steps.text >= (goals.steps || 0) / 2) {
+                steps.style.fill = 'fb-peach'; //yellow
+            } else {
+                steps.style.fill = 'fb-orange'; //pink
+            }
+        } catch (e) {
```

Comment:
Make sure you test all your try statements by forcing the error
`throw "error message of your choice"` in the first line of your try. For this one it will crash the app as you defined the `catch(e)` and then have `err` in your body text output

Reply:
Keep it conistent. Either `catch(err)` or `catch(e)`